tools/go-cli/go-whisk-cli/wski18n/resources/en_US.all.json

```diff
@@ -0,0 +1,1066 @@
+    "translation": "Invalid --shared argument value '{{.argval}}'; valid values are 'yes' or 'no'"
+  },
+  {
+    "id": "Unable to create trigger '{{.name}}': {{.err}}",
+    "translation": "Unable to create trigger '{{.name}}': {{.err}}"
+  },
+  {
+    "id": "{{.ok}} created trigger {{.name}}\n",
+    "translation": "{{.ok}} created trigger {{.name}}\n"
+  },
+  {
+    "id": "update existing trigger",
+    "translation": "update existing trigger"
+  },
+  {
+    "id": "Unable to update trigger '{{.name}}': {{.err}}",
+    "translation": "Unable to update trigger '{{.name}}': {{.err}}"
+  },
+  {
+    "id": "{{.ok}} updated trigger {{.name}}\n",
+    "translation": "{{.ok}} updated trigger {{.name}}\n"
+  },
+  {
+    "id": "get trigger",
+    "translation": "get trigger"
+  },
+  {
+    "id": "Unable to get trigger '{{.name}}': {{.err}}",
+    "translation": "Unable to get trigger '{{.name}}': {{.err}}"
+  },
+  {
+    "id": "trigger /{{.namespace}}/{{.name}}\n",
+    "translation": "trigger /{{.namespace}}/{{.name}}\n"
+  },
+  {
+    "id": "{{.ok}} got trigger {{.name}}\n",
+    "translation": "{{.ok}} got trigger {{.name}}\n"
+  },
+  {
+    "id": "delete trigger",
+    "translation": "delete trigger"
+  },
+  {
+    "id": "Unable to delete trigger '{{.name}}': {{.err}}",
+    "translation": "Unable to delete trigger '{{.name}}': {{.err}}"
+  },
+  {
+    "id": "{{.ok}} deleted trigger {{.name}}\n",
+    "translation": "{{.ok}} deleted trigger {{.name}}\n"
+  },
+  {
+    "id": "list all triggers",
+    "translation": "list all triggers"
+  },
+  {
+    "id": "Unable to obtain the trigger list for namespace '{{.name}}': {{.err}}",
+    "translation": "Unable to obtain the trigger list for namespace '{{.name}}': {{.err}}"
+  },
+  {
+    "id": "Unable to invoke trigger '{{.trigname}}' feed action '{{.feedname}}'; feed is not configured: {{.err}}",
+    "translation": "Unable to invoke trigger '{{.trigname}}' feed action '{{.feedname}}'; feed is not configured: {{.err}}"
+  },
+  {
+    "note-to-translators": "DO NOT TRANSLATE THE 'yes' AND 'no'.  THOSE ARE FIXED CLI ARGUMENT VALUES",
+    "id": "trigger visibility `SCOPE`; yes = shared, no = private",
+    "translation": "trigger visibility `SCOPE`; yes = shared, no = private"
+  },
+  {
+    "id": "trigger feed `ACTION_NAME`",
+    "translation": "trigger feed `ACTION_NAME`"
+  },
+  {
+    "id": "summarize trigger details",
+    "translation": "summarize trigger details"
+  },
+  {
+    "id": "parameter values in `KEY VALUE` format",
+    "translation": "parameter values in `KEY VALUE` format"
+  },
+  {
+    "id": "Arguments must be comma separated, and must be quoted if they contain spaces.",
+    "translation": "Arguments must be comma separated, and must be quoted if they contain spaces."
+  },
+  {
+    "id": "Invalid arguments: {{.err}}",
+    "translation": "Invalid arguments: {{.err}}"
+  },
+  {
+    "id": "The argument `{{.arg}}` is invalid: {{.err}}",
```

Comment:
@mdeuser, Seems to be a number of inconsistencies between messages containing tick marks around arguments instead of single quotes.